bower.json

```diff
@@ -43,7 +43,7 @@
     "dagre": "https://raw.githubusercontent.com/cpettitt/dagre/v0.7.1/dist/dagre.js",
     "d3-cloud": "~1.2.0",
     "amcharts3": "~3.17.0",
-    "simpleheat": "git://github.com/mourner/simpleheat.git#gh-pages"
+    "simpleheat": "https://mourner.github.io/simpleheat/simpleheat.js"
```

Comment:
This is not the correct URL - if you rebase this on my PR here:  https://github.com/hpcc-systems/Visualization/pull/1109 it will fix your travis issues.